prospector/datamodel/advisory_test.py

```diff
@@ -3,7 +3,12 @@
 from unittest import result
 
 from pytest import skip
-from datamodel.advisory import AdvisoryRecord, build_advisory_record
+import pytest
+from datamodel.advisory import (
+    LOCAL_NVD_REST_ENDPOINT,
```

Comment:
(minor comment) I an not sure that LOCAL_NVD_REST_ENDPOINT should be in datamodel.advisory